_specs/ecip-1001.md

```diff
@@ -0,0 +1,86 @@
+---
+lang: en
+ecip: 1001
+title: ECIP Code of Conduct
+status: Draft
+type: Meta
+discussions-to: https://github.com/ethereumclassic/ECIPs/issues/238
+author: Yaz Khoury (@YazzyYaz)
+created: 2019-12-03
+license: CC-By-4.0 License
+---
+This Code of Conduct ECIP has been taken directly from [Contributor Covenant Open Source Code of Conduct](https://www.contributor-covenant.org/)
+under the [CC-By-4.0 License](https://github.com/ContributorCovenant/contributor_covenant/blob/release/LICENSE.md)
+
+This Code of Conduct ECIP applies to all members wishing to participate in the ECIP process:
+1) ECIP Editors
+2) ECIP Coordinators
+3) Hardfork Coordinators
+4) ECIP Authors
+5) Ethereum Classic Community Members Participating in the ECIP Process.
+
+
+## Our Pledge
+We as members, contributors, and leaders pledge to make participation in our community a harassment-free experience for everyone, regardless of age, body size, visible or invisible disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+We pledge to act and interact in ways that contribute to an open, welcoming, diverse, inclusive, and healthy community.
+
+## Our Standards
+Examples of behavior that contributes to a positive environment for our community include:
+
+* Demonstrating empathy and kindness toward other people
+* Being respectful of differing opinions, viewpoints, and experiences
+* Giving and gracefully accepting constructive feedback
+* Accepting responsibility and apologizing to those affected by our mistakes, and learning from the experience
+* Focusing on what is best not just for us as individuals, but for the overall community
+
+Examples of unacceptable behavior include:
+
+* The use of sexualized language or imagery, and sexual attention or advances of any kind
+* Trolling, insulting or derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others’ private information, such as a physical or email address, or real names, without their explicit permission
```

Comment:
IMO this should be the only thing on this list

Reply:
That's fair. What about harassment?

Reply:
Playing the devil's advocate here:
What if someone does something other than doxing, say SWATing, deep-fakes of loved ones, etc. because they don't like an ECIP you proposed. 

That won't be covered by doxing but should be covered by harassment.

I think trolling is fine if its just for shit and giggles. Personal attacks I'm not sure about since they can be very subjective. Say we remove them then until we have a better definition. Political attacks I feel have been going on for a while in the community (calling someone a communist or a stupid liberal, etc.) I just don't know what one should do in this case. We can remove it until we also have a better definition of it.

We should be able to leave room for dissent and instigation since it's what keeps our process honest. Maybe you can provide a draft for something you have in mind in regards to that?